docs/pages/contributing/documentation/style-guide.mdx

```diff
@@ -5,108 +5,152 @@ description: Use this reference to ensure our docs are consistent and effective
+following basic information model:
+
+- Tutorials are learning-oriented and designed to give a newcomer a positive, successful 
+  experience. Tutorials should be tested regularly to ensure they don't get stale and that
+  they always provide a successful result.
+- How-to guides are task-oriented with practical steps to achieve a specific goal. 
+  These guides assume that readers are familiar enough with Teleport that they know the task
+  they want to complete for a specific scenario but not necessarily how to complete it.
+- References are information-oriented technical descriptions of a system or its components.
+- Explanations are the conceptual framework for understanding how or why a system works.
+  Explanation provide the context that surrounds what people learn in a tutorial, do in a 
+  how-to guide, or look up in reference material.
+
+For more detailed information and examples of how to structure documentation using this 
+information model, see [The Grand Unified Theory of Documentation](https://documentation.divio.com/).
+
+### Structuring content into pages
+
+In general, every page of documentation should have a purpose and be self-contained. 
+It's okay for information to be in multiple places and similar content can exist in 
+pages that serve different purposes. For example, you might have conceptual information
+about trusted clusters in one page and have similar content as context in a how-to topic.
+When possible, you should reuse content rather than duplicate content, but it's perfectly
+fine to cover the same information in more than one place to serve different purposes.
+
+## How-to guides
+
+A how-to guide describes how to achieve a goal or complete a task. 
+Only the information that is pertinent to achieving that goal or completing the task is included. 
+With how-to guides, readers have enough information to know what they want to do—for example, 
+open a bank account—but not necessarily enough information to know how to do it.
+For example, the how-to guide for opening a bank account wouldn't explain what a bank account 
+is or why you might want to open one, but would focus on specific steps such as:
+
+1. Select an institution.
+1. Fill out an application.
+1. Deposit a minimum amount of currency.
+ 
+How-to guides often include links to additional information, but should not include 
+explanations that take the focus away from what the reader wants to accomplish.
+
+### What's in a how-to guide?
+
+In most cases, how-to guides contain the following sections:
+
+- Frontmatter with a guide title, description, and other information.
+  The title should be short and identify the subject of the how-to topic in the fewest words possible.
+  The description should be a sentence that starts with a verb and summarizes the content of the topic.
+  Additional information might include a video banner link, a list of keywords, or an alternate 
+  first-level heading.
```

Comment:
This applies to docs pages in general—and is specific to our docs engine—rather than the "how-to guide" genre. I would move this somewhere else.

Also, I'm curious as to the reasoning behind this recommendation: 

>   The description should be a sentence that starts with a verb and summarizes the content of the topic.